defold-rive/pluginsrc/rive_file.cpp

```diff
@@ -298,14 +301,55 @@ static void Render(RiveFile* rive_file)
         dmRive::ApplyDrawMode(ro, draw_desc.m_DrawMode, draw_desc.m_ClipIndex);
 
         memcpy(&ro.m_WorldTransform, &vs_uniforms.world, sizeof(vs_uniforms.world));
+
+        switch(draw_desc.m_DrawMode) {
+            case DRAW_MODE_DEFAULT:
+                dmLogInfo("  DRAW_MODE_DEFAULT");
+                break;
+            case DRAW_MODE_CLIP_DECR:
+                dmLogInfo("  DRAW_MODE_CLIP_DECR");
+                break;
+            case DRAW_MODE_CLIP_INCR:
+                dmLogInfo("  DRAW_MODE_CLIP_INCR");
+                break;
+        }
```

Comment:
Do we really want to have this log spam in the render function?

Reply:
Eeeh oops, no!